utils/tini/patches/002-fix_missing_basename.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/tini.c
++++ b/src/tini.c
+@@ -18,6 +18,8 @@
+ #include "tiniConfig.h"
+ #include "tiniLicense.h"
+ 
++#define basename(name) (strrchr((name),'/') ? strrchr((name),'/')+1 : (name))
```

Comment:
One issue with this is that glibc defines basename in string.h, which is included in tini. This define would conflict.

Reply:
Ok, thanks a lot. Close